### PR TITLE
Cleanup cabal file

### PR DESCRIPTION
**Pull request by jhrcek**
Few additional changes that got lost in the merge ..

### CODE REVIEW

File: postgresql-migration.cabal [hunk 1]
```diff
@@ -12,13 +12,15 @@ copyright:                  2014-2021, Andreas Meingast
...
-
-extra-source-files:         License
-                            Readme.markdown
+tested-with:                GHC==9.6.1
+                            GHC==9.4.4
+                            GHC==9.2.7
+                            GHC==9.0.2
+                            GHC==8.8.4
+                            GHC==8.6.5
+extra-doc-files:            Readme.markdown
```
Comment (jhrcek) [inline postgresql-migration.cabal:21]: `cabal check` tell me Readme and Changelog should live under `extra-doc-files`.
Licence shouldn't be added as `extra-source-file` - it's already present under `license-file` above.

File: postgresql-migration.cabal [hunk 2]
```diff
@@ -30,31 +32,32 @@ common common-options
     base >=4.9 && <5
   default-language:
     Haskell2010
+  ghc-options: -Wall
```
Comment (jhrcek) [inline postgresql-migration.cabal:35]: Consolidate all common options under one common-options - as all of library, tests and executable were using identical options.

**Comment by jhrcek** [comment]
Hello @andrevdm
Could you please check/merge these simplifications?
This should simplification should make future maintenance easier :slightly_smiling_face:

**Comment by andrevdm** [comment]
Merged, and thank you

**Pull request merged**